model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -49,42 +49,48 @@ private fun createCopyrightFinding(statement: String, path: String, line: Int) =
         )
     )
 
-class FindingsMatcherTest : WordSpec({
-    val matcher = FindingsMatcher(LicenseFileMatcher("a/LICENSE"))
+class FindingsMatcherTest : WordSpec() {
+    companion object {
+        private const val LICENSE_FILE_PATH = "a/LICENSE"
```

Comment:
Hmm. I find it odd to have a constant for "a/LICENSE" now, but not for "b/LICENSE".

Generally, I have to admit I dislike the use of such constants in tests, because for me tests should put an emphasis on readability of what the test does / compares, and hiding literal strings behind constants does not help here. 

Finally, constants make sense to be used in places where the string really *has* to be the same, so that changing the constant changes the string in all places. But here, the path strings used in the tests *could* be different, and we just happen to be using the same string out of laziness. So there is no technical requirement for these strings to be the same, which for me is an argument to not hide them behind a constant.

Reply:
> Hmm. I find it odd to have a constant for "a/LICENSE" now, but not for "b/LICENSE".

Have to admit that it is a bit odd yes. The idea of this change anyhow is that LICENSE_FILE_PATH denotes the license file passed to the ctor of the LICENSE_FILE_PATH  whereas "b/LICENSE" is a string pointing to any other path. I propose to fix this by changing the value of "b/LICENSE" to e.g. "some/file/path". Would that make sense for you as well ?

> Finally, constants make sense to be used in places where the string really has to be the same

Isn't this the case here? ..I'm asking as the string is used in two locations and it has to be those have to be the same.